LICENSE

```diff
@@ -0,0 +1,504 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
I have no idea which GNU license to use.  I see a variety of licenses in NOAA-EMC repositories.   Should we have consistency across NOAA-EMC repositories?